code/parachain/runtime/common/src/fees.rs

```diff
@@ -105,16 +98,24 @@ impl<AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>>
 	) -> Result<Balance, Self::Error> {
 		AssetsRegistry::get_ratio(asset_id)
 			.and_then(|x| safe_multiply_by_rational(native_amount, x.n().into(), x.d().into()).ok())
-			.or_else(|| {
-				WellKnownForeignToNativePriceConverter::to_asset_balance(native_amount, asset_id)
-			})
+			.or_else(|| ForeignToNative::to_asset_balance(native_amount, asset_id))
 			.ok_or(DispatchError::Other(cross_chain_errors::ASSET_PRICE_NOT_FOUND))
 	}
 }
 
 #[cfg(test)]
 mod commons_sense {
-	use crate::fees::WellKnownForeignToNativePriceConverter;
+	use composable_traits::rational;
+	struct Dummy {}
+	impl ForeignToNativePriceConverter for Dummy {
+		fn get_ratio(asset_id: CurrencyId) -> Option<Rational64> {
+			match asset_id {
+				CurrencyId::USDT | CurrencyId::USDC => Some(rational!(15 / 1_000_000_000)),
+				CurrencyId::KSM => Some(rational!(375 / 1_000_000)),
```

Comment:
Do we add LAYR and DOT here too?

Reply:
that is just for test. that ratios are sane.